.gitmodules

```diff
@@ -25,11 +25,11 @@
 [submodule "mongodb_exporter"]
 	path = sources/mongodb_exporter/src/github.com/percona/mongodb_exporter
 	url = https://github.com/percona/mongodb_exporter.git
-	branch = release-0.20.6
+	branch = release-0.20.8
 [submodule "postgres_exporter"]
 	path = sources/postgres_exporter/src/github.com/percona/postgres_exporter
 	url = https://github.com/percona/postgres_exporter
-	branch = master
+	branch = release-0.4.8
```

Comment:
https://github.com/percona/postgres_exporter/pull/51

this probably should be merged to master and master stay ?